cucumber/rotators/features/support/rotator_helpers.rb

```diff
@@ -4,14 +4,15 @@
 #
 require 'aws-sdk-iam'
 require 'cucumber/policy/features/support/client'
+require 'cucumber/policy/features/support/policy_helpers'
```

Comment:
Will try to remove this one too tomorrow

Reply:
Thanks!

Reply:
Resolved by @sashaCher